config/config.yaml

```diff
@@ -590,6 +590,16 @@ defaults:
       serviceAccountName: clusters-service
     azureRuntimeConfig:
       tlsCertificatesIssuer: OneCertV2-PublicCA
+    # OCP Versions configuration
```

Comment:
If we want to specify this in different envs this can be overwritten in the corresponding env in this file?

Reply:
Yes that is the idea.